src/domain/validations.ts

```diff
@@ -26,17 +34,21 @@ export function validateRepresentativeSection(app: Application) {
   }
   const validations = [
     validatePersonalInfo(app.sections.representative.info, errors, false),
-    validatePrimaryAffiliationMatching(app.sections.representative.info.primaryAffiliation, app.sections.applicant.info.primaryAffiliation, errors)
+    validatePrimaryAffiliationMatching(
+      app.sections.representative.info.primaryAffiliation,
+      app.sections.applicant.info.primaryAffiliation,
+      errors,
+    ),
   ];
 
-  const isValid = addressResult && !validations.some(x => x === false);
-  return {isValid, errors};
+  const isValid = addressResult && !validations.some((x) => x === false);
+  return { isValid, errors };
 }
 
 export function validateApplicantSection(app: Application) {
   const applicantErrors: SectionError[] = [];
   const addressResult = validateAddress(app.sections.applicant.address, applicantErrors);
-  const infoResult = validatePersonalInfo(app.sections.applicant.info, applicantErrors);
+  const infoResult = validatePersonalInfo(app.sections.applicant.info, applicantErrors, true, true);
```

Comment:
Wish JS has a way to pass descriptive booleans, so we can just read the code without relying on VSCode's tooltips (which tell you the signature of the function etc.)

For these cases, what I tend to do is pass a coerced string (e.g. `!!'validateWebsiteRequired'`), but it doesn't look pretty at all.

Reply:
yeah with typescript plain boolean args don't bother me so much as it's at least easy to track down the intention